platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBook.java

```diff
@@ -178,7 +178,7 @@ public long getClassId() {
      * @return the round when this address book was constructed, or {@link #UNKNOWN_ROUND} if the round is unknown or
      * this address book was not constructed during a regular round
      */
-    public long getRound() {
+    public long getRound() { // TODO remove
```

Comment:
Is this meant to be future work?  Is the method deprecated?

Reply:
Should not be done in this PR. TODO has been removed.